<launch>

  <arg name="network" default="sarge"/>
  <arg name="ranging_radios" default="false"/>

  <!-- global params -->
  <param name="/scan_range_min" value="0.45"/>
  <param name="/scan_range_max" value="6.0"/>

  <node pkg="rviz" type="rviz" name="rviz" 
    args="-d $(find csqmi_exploration)/rviz/1robot.rviz" />

  <!-- Robot 1 -->
  <machine name="robot1" address="192.168.1.151" user="odroid"
    env-loader="~/catkin_ws/src/robot_io/launch/env/$(arg network)_env_loader.zsh"
    default="true"/>
  <include file="$(find robot_io)/launch/mast.launch">
      <arg name="tf_prefix"       value="robot1"/>
      <arg name="radio_id"        value="101"/>
      <arg name="ranging_radio"   value="$(arg ranging_radios)"/>
  </include>
  
  <!-- Robot 2
  <machine name="robot2" address="192.168.0.152" user="odroid"
    env-loader="~/catkin_ws/src/robot_io/launch/env/$(arg network)_env_loader.zsh"
    default="true"/>
  <include file="$(find robot_io)/launch/mast.launch">
      <arg name="tf_prefix"       value="robot2"/>
      <arg name="radio_id"        value="100"/>
      <arg name="ranging_radio"   value="$(arg ranging_radios)"/>
  </include>
 -->
  <!-- Robot 3
  <machine name="robot3" address="192.168.0.153" user="odroid"
    env-loader="~/catkin_ws/src/robot_io/launch/env/$(arg network)_env_loader.zsh"
    default="true"/>
  <include file="$(find robot_io)/launch/mast.launch">
      <arg name="tf_prefix"       value="robot3"/>
      <arg name="radio_id"        value="104"/>
      <arg name="ranging_radio"   value="$(arg ranging_radios)"/>
  </include>
 -->
</launch>
